### PR TITLE
lint: fix for pubsub, pubsub_grpc, runtime, service_invocation_test e2e tests

### PR DESCRIPTION
**Pull request by jigargandhi**
# Description

Fixed lint for pubsub, pubsub_grpc, runtime, service invocation tests

## Issue reference

Please reference the issue this PR will close: #3328 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/e2e/pubsub_grpc/pubsub_grpc_test.go [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // Licensed under the MIT License.
 // ------------------------------------------------------------
 
-package pubsubapp_e2e
```
Comment (artursouza) [inline tests/e2e/pubsub_grpc/pubsub_grpc_test.go:8]: can we keep the e2e in the namespace somehow?

Comment (jigargandhi) [reply tests/e2e/pubsub_grpc/pubsub_grpc_test.go:8]: I think that would not be possible if we want to write [Effective Go](https://golang.org/doc/effective_go), where underscores, and mixedCaps are not recommended. 

I took a look at other projects to see how they solve this naming problem and found that they are not including `e2e` in package name 
Example 
- nomad  https://github.com/hashicorp/nomad/blob/main/e2e/connect/connect.go
- kubernetes https://github.com/kubernetes/kubernetes/blob/master/test/e2e/auth/certificates.go

Comment (daixiang0) [reply tests/e2e/pubsub_grpc/pubsub_grpc_test.go:8]: Consider that we do not call those packages outside e2e, remove suffix is safe and make it clear.

**Comment by daixiang0** [comment]
@jigargandhi please fix conflict

**Review by artursouza — APPROVED** [review]

**Pull request merged**